### PR TITLE
Unify spelling of controllers

### LINKED ISSUE #3814: Unify spelling of controllers
In `Modelica.Blocks.Continuous` the controllers are spelled, for example, the following way:

 - PI-controller
 - PID-controller

In Electropedia the following spelling is used:

- [PI element](https://www.electropedia.org/iev/iev.nsf/display?openform&ievref=351-50-11) or *proportional plus integral element*
- [PID element](https://www.electropedia.org/iev/iev.nsf/display?openform&ievref=351-50-19) or *proportional plus integral plus derivative element*

From my understanding it would thus also makes sense to omit the dash when using the term *controller* instead of *element*:

- PI controller or *proportional plus integral controller*
- PID controller or *proportional plus integral plus derivative controller*

A comment from a native English speaking person is highly appreciated here. Possibly @mtiller or @harmanpa could comment here.

Once decided, this shall be documented in `Modelica.UsersGuide.Conventions.Terms.Electrical`.

### PR DESCRIPTION
**Pull request by arunkumar-narasimhan**
Close #3814. unify spelling of controllers by omitting dash in description and documentation

### CODE REVIEW

**Review by christiankral — APPROVED** [review]
Looks good!

File: Modelica/Blocks/Continuous.mo [hunk 1]
```diff
@@ -1011,18 +1011,18 @@ together) and using the following strategy:
...
      <strong>k</strong> and <strong>Ti</strong> (the time constant of the integrator).
      The first value of Ti can be selected, such that it is in the
      order of the time constant of the oscillations occurring with
-     the P-controller. If, e.g., vibrations in the order of T=10 ms
+     the P controller. If, e.g., vibrations in the order of T=10 ms
      occur in the previous step, start with Ti=0.01 s.</li>
 <li> If you want to make the reaction of the control loop faster
      (but probably less robust against disturbances and measurement noise)
-     select a <strong>PID</strong>-Controller and manually adjust parameters
+     select a <strong>PID</strong> Controller and manually adjust parameters
```
Comment (tobolar) [inline Modelica/Blocks/Continuous.mo:1025]:
```suggestion
     select a <strong>PID</strong> controller and manually adjust parameters
```

File: Modelica/Clocked/Examples/Systems/ControlledMixingUnit.mo [hunk 2]
```diff
@@ -249,7 +249,7 @@ as desired cooling temperature at the output of the controller)
 and the desired temperature T (which is used as desired value for
 the feedback controller). This part of the control system is the
 \"feed-forward\" part that computes the desired actuator signal.
-As feedback controller a simple P-Controller with one gain is used.
+As feedback controller a simple P Controller with one gain is used.
```
Comment (tobolar) [inline Modelica/Clocked/Examples/Systems/ControlledMixingUnit.mo:252]:
```suggestion
As feedback controller a simple P controller with one gain is used.
```

**Comment by tobolar** [comment]
Please consider my comment https://github.com/modelica/ModelicaStandardLibrary/issues/3814#issuecomment-1558725707

**Comment by arunkumar-narasimhan** [comment]
> Please consider my comment [#3814 (comment)](https://github.com/modelica/ModelicaStandardLibrary/issues/3814#issuecomment-1558725707)

@tobolar, we have made changes per your comment [#3814](https://github.com/modelica/ModelicaStandardLibrary/issues/3814#issuecomment-1558725707)

**Review by tobolar — APPROVED** [review]
Fine to me.

**Pull request merged**